bootstrap.js

```diff
@@ -9,25 +9,31 @@ const ResponseBuilder = require('./app/ResponseBuilder');
 module.exports = app => {
     app.get('/*', (req, res) => {
         const responseBuilder = new ResponseBuilder(res);
-        
+
         const requestedUrl = req.url.slice(1);
         const corsBaseUrl = '//' + req.get('host');
-        
+
         console.info(req.protocol + '://' + req.get('host') + req.url);
-        
+
         if(requestedUrl == ''){
             res.send(index);
             return;
         }
 
+        let headers = Object.assign({}, req.headers);
+
+        // This prevents "Error [ERR_TLS_CERT_ALTNAME_INVALID]: Hostname/IP does not match certificate's altnames: Host: localhost"
+        delete headers.host;
+
+        // This prevents "Received response with content-encoding: gzip, but failed to decode it"
+        delete headers['accept-encoding'];
+
         request({
             uri: requestedUrl,
             resolveWithFullResponse: true,
-            headers: {
-                'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36'
```

Comment:
Could we ensure this user agent is still passed through on the headers object, looks at present that this has been removed?

Reply:
If the target system depends on the user agent being set properly, having the proxy overwriting it with something else will just break things.

To me this behavior was really unexpected; maybe document the rationale for this?